### PR TITLE
Program: GCI Back button improper behavior

### PR DESCRIPTION
**Pull request by ValentunSergeev**
#503

### CODE REVIEW

**Comment by ValentunSergeev** [comment]
@Rimjhim28 I've updated the PR:
* Added back behavior to MapActivity
* Removed BaseActivity class

File: PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java [hunk 1]
```diff
@@ -193,6 +192,32 @@ private void updatePoints(int position) {
         SessionHistory.totalPoints += answers.get(position).getPoints();
     }
```
Comment (codingblazer) [inline PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java]: Hey @ValentunSergeev, your work is perfect considering what was required. But you have added dialog box which was not needed. Please remove dialog later when you have time (maybe after gci). I am approving your task for now.

Comment (ValentunSergeev) [reply PowerUp/app/src/main/java/powerup/systers/com/GameActivity.java]: Hi @codingblazer I removed this dialog

**Comment by m-sameer** [comment]
@ValentunSergeev @codingblazer Pressing back from **MapActivity** and **ScenarioOverActivity** will close the app instead of directly the user to **StartActivity** and **MapActivity**, respectively. :)

**Pull request closed without merging**